benchmarks/src/jmh/java/software/amazon/jdbc/benchmarks/PluginBenchmarks.java

```diff
@@ -162,21 +215,17 @@ public ResultSet executeStatementWithExecutionTimePlugin() throws SQLException {
     }
   }
 
-  Properties useAllPlugins() {
-    final Properties properties = new Properties();
-    properties.setProperty("proxyDriverPlugins", "executionTime,auroraHostList");
-    return properties;
-  }
-
-  Properties useExecutionPlugin() {
+  Properties usePlugins(String propsStr) {
     final Properties properties = new Properties();
-    properties.setProperty("proxyDriverPlugins", "executionTime");
+    properties.setProperty("wrapperPlugins", propsStr);
     return properties;
   }
 
-  Properties useAuroraHostList() {
+  Properties usePluginsWithReaderLoadBalancing(String propsStr) {
```

Comment:
I'd prefer to see different methods instead of a single one that accepts plugin name parameter.

Reply:
In other words, lets don't change any existing methods but just add new ones if needed.